packages/synapse-react-client/src/components/CitationPopover/CitationPopover.tsx

```diff
@@ -0,0 +1,261 @@
+  Button,
+  Popover,
+  Typography,
+  Box,
+  Stack,
+  FormControl,
+  Select,
+  MenuItem,
+  SelectChangeEvent,
+  Alert,
+  SxProps,
+} from '@mui/material'
+import CopyToClipboardIcon from '../CopyToClipboardIcon'
+import CloseIcon from '@mui/icons-material/Close'
+import { KeyboardArrowDown } from '@mui/icons-material'
+import { useCitation } from './useCitation'
+import { ReactComponent as DoubleQuotes } from '../../assets/icons/DoubleQuotes.svg'
+import { createLinkAndDownload } from './CitationPopoverUtils'
+
+type CitationPopoverProps = {
+  doi: string | undefined
+  title?: string
+  boilerplateText?: string
+}
+
+const selectSx: SxProps = {
+  width: '100%',
+  '.MuiInputBase-root': {
+    minHeight: '38px',
+    borderRadius: 0,
+    '&:before, &:after': {
+      borderBottom: 'none !important',
+    },
+  },
+  '& .MuiOutlinedInput-notchedOutline': {
+    border: 0,
+  },
+  '&.MuiOutlinedInput-root.Mui-focused .MuiOutlinedInput-notchedOutline': {
+    border: 'none',
+  },
+  '.MuiSelect-select': {
+    display: 'flex',
+    alignItems: 'center',
+  },
+  '.MuiInputBase-input': {
+    padding: '7px 10px',
+  },
+}
+
+function CitationPopover({
+  doi,
+  title,
+  boilerplateText,
+}: CitationPopoverProps) {
+  const [anchorEl, setAnchorEl] = useState<HTMLButtonElement | null>(null)
+  const [citationFormat, setCitationFormat] = useState('bibtex')
+
+  const handleChange = (event: SelectChangeEvent) => {
+    setCitationFormat(event.target.value)
+  }
+
+  const handleClick = (event: React.MouseEvent<HTMLButtonElement>) => {
+    setAnchorEl(event.currentTarget)
+  }
+
+  const handleClose = () => {
+    setAnchorEl(null)
+  }
+
+  const handleDownload = (citation: string) => {
+    if (!citation) {
+      return
+    }
+    const extensionMap: Record<string, string> = {
+      apa: 'ris',
+      bibtex: 'bib',
+      ieee: 'ris',
+      nature: 'ris',
+      science: 'ris',
+    }
+    const fileExtension = extensionMap[citationFormat] || 'txt'
+
+    const blob = new Blob([citation], { type: 'text/plain' })
+    const url = URL.createObjectURL(blob)
+
+    createLinkAndDownload(title || 'citation', fileExtension, url)
+  }
+
+  const open = Boolean(anchorEl)
+  const id = open ? 'cite-as-popover' : undefined
+
+  const {
+    data: citation,
+    isLoading,
+    error,
+  } = useCitation(doi || '', citationFormat, open)
+
+  return (
+    <div>
+      {doi && (
+        <Button
+          onClick={handleClick}
+          variant="outlined"
+          startIcon={<DoubleQuotes />}
+          sx={{
+            display: 'flex',
+            height: '20px',
+            padding: '2px 8px',
+            borderColor: 'grey.400',
+            '& .MuiButton-startIcon': { marginRight: '4px', marginLeft: 0 },
+          }}
+        >
+          <Typography variant="label" sx={{ lineHeight: '20px' }}>
+            Cite As
+          </Typography>
+        </Button>
+      )}
+      <Popover
+        aria-label="Citation options popover"
```

Comment:
This is redundant, the accessibility device (screen reader) will already see that this is a dialog
```suggestion
        aria-label="Citation options"
```